lib/elixir/lib/code.ex

```diff
@@ -50,9 +50,11 @@ defmodule Code do
 
   ## Examples
 
-      Code.append_path(".") #=> true
+      iex> Code.append_path(".")
```

Comment:
this line (and similar) has side effects so I'm not 100% sure if it won't mess with other tests